runtime/queries/column_timeseries.go

```diff
@@ -131,31 +139,35 @@ func (q *ColumnTimeseries) Resolve(ctx context.Context, rt *runtime.Runtime, ins
 		}()
 
 		rows, err := olap.Execute(ctx, &drivers.Statement{
-			Query:    `SELECT * from "` + temporaryTableName + `"`,
+			Query:    fmt.Sprintf(`SELECT %s as ts, * EXCLUDE(%s) FROM %s`, tsAlias, tsAlias, temporaryTableName),
```

Comment:
There can be `ts` measure name in the temporary table if the user names a measure that way, see the last `select`:
```
			SELECT
			` + getCoalesceStatementsMeasures(measures) + `,
```

Reply:
Ahh, I didn't know DuckDB could have multiple result columns with same name! But that's pretty weird behavior (don't think many other DBs would allow that), imagine if reading that into e.g. a map. It will also get messed up in JSON transaltion. Let's avoid that